src/dist/docbook-xsl/fo-pdf.xsl

```diff
@@ -48,11 +48,11 @@
   -->
 
   <xsl:template name="pickfont-sans">
-    <xsl:text>Arial,sans-serif</xsl:text>
+    <xsl:text>Liberation Sans,Arial,sans-serif</xsl:text>
   </xsl:template>
 
   <xsl:template name="pickfont-serif">
-    <xsl:text>Georgia,serif</xsl:text>
+    <xsl:text>DejaVu Serif,Liberation Serif,serif</xsl:text>
```

Comment:
The Microsoft font license is one of the reasons why MS still has a quasi monopole in business word processing. I believe that asciidoctor should not tie its user to third party licenses. 
If you agree with me please have a look at the [excellent alternatives](https://en.wikipedia.org/wiki/Arial#Free_alternatives) .
For print my personal favourites are the [DejaVu fonts](https://en.wikipedia.org/wiki/DejaVu_fonts) but the [Liberation Fonts](https://en.wikipedia.org/wiki/Liberation_fonts) are of excellent quality also. The latter have exactly the same metrics than MS fonts, most end users would not notice a difference. 
Please have a look. This is the right moment for liberation as the layout changes anyway?  ;)